Solutions/PingFederate/Data Connectors/Connector_CEF_PingFederate.json

```diff
@@ -0,0 +1,121 @@
+{
+    "id": "PingFederate",
+    "title": "PingFederate",
+    "publisher": "Ping Identity",
+    "descriptionMarkdown": "The [PingFederate](https://www.pingidentity.com/en/software/pingfederate.html) data connector provides the capability to ingest [PingFederate events](https://docs.pingidentity.com/bundle/pingfederate-102/page/lly1564002980532.html) into Azure Sentinel. Refer to [PingFederate documentation](https://docs.pingidentity.com/bundle/pingfederate-102/page/tle1564002955874.html) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **PingFederateEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-PingFederate-parser)  ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "PingFederateEvent",

```

Comment:
Datatype name is PingFederate. So please update legend name to PingFederate.

Thanks,
Aditi Maheshwari